http/src/main/java/io/micronaut/http/HttpMessageWrapper.java

```diff
@@ -69,4 +70,9 @@ public <T> Optional<T> getBody(Class<T> type) {
     public <T> Optional<T> getBody(Argument<T> type) {
         return delegate.getBody(type);
     }
+
+    @Override
+    public <T> Optional<T> getBody(ArgumentConversionContext<T> conversionContext) {
```

Comment:
missing `@NonNull` annotation in return type and in the parameter `conversionContext`